README.md

```diff
@@ -728,6 +728,83 @@ _You can enable the following settings in Xcode by running [this script](resourc
   let dictionary: [String: Any] = [:]
   ```
 
+* <a id='single-line-statement-braces'></a>(<a href='#single-line-statement-braces'>link</a>) The opening brace following a single-line statement should be on the same line as the rest of the statement. [![SwiftFormat: braces](https://img.shields.io/badge/SwiftFormat-braces-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#braces)
```

Comment:
Asking to learn: is there an official definition of a "statement" in Swift?

Reply:
[Statements](https://docs.swift.org/swift-book/ReferenceManual/Statements.html#) and [expressions](https://docs.swift.org/swift-book/ReferenceManual/Expressions.html) have different meanings in the Swift Grammar. I think an "expression" specifically refers to something that can be the left-hand-side of an assignment operator.

Reply:
From looking at these resources (thanks!) `expression` feels more appropriate in this case. As an example, it seems like an "if statement" includes all of the code starting with `if` and ending at the final `}`. If that's true, most if statements would not be a single line unless you had something like `if myBoolean { print("hello" }`.

<img width="783" alt="Screen Shot 2021-06-15 at 7 09 40 PM" src="https://user-images.githubusercontent.com/1791049/122147250-5facf800-ce0d-11eb-919a-68cd876e349f.png">

Reply:
Good point -- I think you're right on this. Updated to refer to these as "expressions".